python/nano/src/bigdl/nano/deps/onnxruntime/__init__.py

```diff
@@ -13,3 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+try:
+    import onnx
+    import onnxruntime as ort
+except ImportError:
+    raise ImportError("To enable onnxruntime inference, you need to install it by:\n"
+                      "\t\t pip install onnxruntime")
```

Comment:
Is this right place to check?

Reply:
Good catch, I should put it to `onnxruntime/core`